src/fixing/Autofix.ml

```diff
@@ -63,6 +63,8 @@ let transform_fix lang ast =
           mk_visitor
             {
               default_visitor with
+              kexpr =
```

Comment:
What does this do?
need a comment. I don't get it.

Reply:
Oh sorry, meant to comment here but forgot... will do.